olive/olive_config.json

```diff
@@ -232,6 +238,9 @@
             "onnxruntime-directml",
             "onnxruntime-gpu",
             "onnxruntime-openvino"
+        ],
+        "nvmo": [
+            "nvidia-modelopt"
```

Comment:
will graphsurgeon always be required for modelopt passes? perhaps we can add it here too?

Reply:
Yes, it will always be required. I will make the change after the current pipeline ends/crashes.